frontend/src/metabase/admin/people/containers/AdminPeopleApp/AdminPeopleApp.tsx

```diff
@@ -0,0 +1,52 @@
+import type * as React from "react";
+import { t } from "ttag";
+
+import { NudgeToPro } from "metabase/admin/people/components/NudgeToPro";
+import { shouldNudgeToPro } from "metabase/admin/people/selectors";
+import { useSetting } from "metabase/common/hooks";
+import { AdminLayout } from "metabase/components/AdminLayout";
+import {
+  LeftNavPane,
+  LeftNavPaneItem,
+  LeftNavPaneSectionTitle,
+} from "metabase/components/LeftNavPane";
+import { useSelector } from "metabase/lib/redux";
+import { Divider, Flex } from "metabase/ui";
+
+export const AdminPeopleApp = ({ children }: { children: React.ReactNode }) => {
+  const shouldNudge = useSelector(shouldNudgeToPro) as boolean;
+  const tenantEnabled = useSetting("use-tenants");
```

Comment:
I'm following the pattern that the BE is using that setting this setting is a EE only feature, but the setting existing is OSS.